gulpfile.js

```diff
@@ -372,6 +372,7 @@ gulp.task('build', gulp.series(clean, 'build-bundle-prod'));
 gulp.task('build-postbid', gulp.series(escapePostbidConfig, buildPostbid));
 
 gulp.task('serve', gulp.series(clean, lint, gulp.parallel('build-bundle-dev', watch, test)));
+gulp.task('serve-fast', gulp.series(clean, gulp.parallel('build-bundle-dev', watch)));
```

Comment:
When I am just trying to test via my E2E test pages having to wait several minutes for lint + test is a little much.

Thought others might use a similar local edit, so why not merge it in.

Reply:
This will come in handy!